src/awx_plugins/credentials/aws_assumerole.py

```diff
@@ -0,0 +1,143 @@
+import datetime
+import hashlib
+
+from django.utils.translation import gettext_lazy as _
+
+import boto3
+
+from .plugin import CredentialPlugin
+
+
+try:
+    from botocore.exceptions import ClientError
```

Comment:
Can we make this a dependency so that it's always importable and wouldn't need to be suppressed?

Reply:
I think we could. My guess is that this was probably in place given how EEs function in AWX but given the context of how we bring in plugins like this, it would have been added to the AWX so this would have been removed at some point if we were not splitting out modules. 

Will add the dep and remove the warning. This also looks like a pattern that might been copied from the AWS collection since I believe they handle their dep requirements in catches like this.